apps/emqx_management/test/emqx_mgmt_api_configs_SUITE.erl

```diff
@@ -283,7 +283,7 @@ t_configs_node({'init', Config}) ->
     ),
     Config;
 t_configs_node({'end', _}) ->
-    meck:unload([emqx, emqx_management_proto_v2]);
+    meck:unload([emqx, emqx_management_proto_v4]);
```

Comment:
nit: should we add some test cases to ensure that the node parameter has taken effect?